diff --git a/nodejs.config.js.example b/nodejs.config.js.example
index 698751c..f5b71a8 100644
--- a/nodejs.config.js.example
+++ b/nodejs.config.js.example
@@ -106,7 +106,7 @@ settings = {
   extensions: [],
   clientsCanWriteToChannels: false,
   clientsCanWriteToClients: false,
-  transports: ['websocket', 'flashsocket', 'htmlfile', 'xhr-polling', 'jsonp-polling'],
+  transports: ['websocket', 'polling'],
   jsMinification: true,
   jsEtag: true,
   logLevel: 1
diff --git a/nodejs_config/nodejs_config.module b/nodejs_config/nodejs_config.module
index 76a5290..9b8571c 100644
--- a/nodejs_config/nodejs_config.module
+++ b/nodejs_config/nodejs_config.module
@@ -187,7 +187,7 @@ function nodejs_config_form_submit($form, &$form_state) {
     'extensions' => $ext,
     'debug' => (bool) $values['nodejs_config_debug'],
     'transports' => array(
-     'websocket', 'flashsocket', 'htmlfile', 'xhr-polling', 'jsonp-polling'
+        'websocket', 'polling'
     ),
    'jsMinification' => true,
    'jsEtag' => true,
diff --git a/server.js b/server.js
index ea5eb6e..c46c542 100644
--- a/server.js
+++ b/server.js
@@ -45,7 +45,7 @@ var channels = {},
       extensions: [],
       clientsCanWriteToChannels: false,
       clientsCanWriteToClients: false,
-      transports: ['websocket', 'flashsocket', 'htmlfile', 'xhr-polling', 'jsonp-polling'],
+      transports: ['websocket', 'polling'],
       jsMinification: true,
       jsEtag: true,
       backend: {
